bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -104,13 +105,70 @@ public void initialize() {
         }
         updateStatus(ThingStatus.UNKNOWN);
 
+        updateState(CHANNEL_HVAC_TARGET_TEMPERATURE,
+                new QuantityType<Temperature>(car.getHvacTargetTemperature().doubleValue(), SIUnits.CELSIUS));
```

Comment:
The idea is to force a value 20°C at init ?

Reply:
Yes. 20 is one of a few values that does not cause problems... I think it only affects pre-heat.  (This is at least true from my testing)

Reply:
Why setting a value at init if you do not what is the real value. This can show a wrong information to users.
This channel is more a "command" channel to set a target temperature. It can be NULL at init and the user will set it to a value when he wants to set a target temperature.

Reply:
I want them to keep the default so I try to force it.